kolibri/auth/apps.py

```diff
@@ -0,0 +1,9 @@
+from __future__ import unicode_literals
```

Comment:
I'd prefer the full py3-compatible future in all our modules...

    from __future__ import absolute_import, print_function, unicode_literals